WordPress/Jetpack/Classes/NUX/JetpackPrologueViewController.swift

```diff
@@ -149,3 +153,9 @@ class JetpackPrologueViewController: UIViewController {
         static let gradientLayerPosition: CGFloat = -99
     }
 }
+
+extension JetpackPrologueViewController: InfiniteScrollerViewDelegate {
+    var rate: CGFloat {
```

Comment:
This is totally optional but I think it would be great to declare `rate` as a method like this:
```swift
func rate(for infiniteScrollerView: InfiniteScrollerView) -> CGFloat
```
The reason is that we may want to add a `rate` property that's private to this view controller.
```swift
private var rate: CGFloat = 0
```
And it's also aligned with how delegate protocols are implemented in UIKit. For example, the `UITableViewDataSource`:
```swift
func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int
func numberOfSections(in tableView: UITableView) -> Int
```

Reply:
I'd personally leave things as simple as possible here, unless we see a specific need to pass a reference of `InfiniteScrollerView` that then gets used in the delegate method (which is the case of the table view delegate/data source methods).

Reply:
I like both of your points @salimbraksa and @Gio2018. I went ahead and updated it in 61d422baee8e78670395256db6b17067b596ed9e. 🙇